sbnci/SBND/PlottingScripts/sbnciplot_pdssimvalidation_sbnd.C

```diff
@@ -7,52 +7,67 @@
 #include "TFile.h"
 #include "TLegend.h"
 #include "TCollection.h"
-#include <iostream>
 #include "THStack.h"
 #include "TAxis.h"
 #include "TGraphErrors.h"
+#include "TString.h"
+
+#include <iostream>
+#include <map>
 #include <sstream>
 #include <vector>
-#include "TString.h"
 
-void PlotHists(TFile *inFile, TFile *outFile, TString treeName, TString histName, TString histTitle, TString histXaxis, TString histYaxis, int nBins, double xMin, double xMax, TString metricToPlot, TString condition, TString dirName);
+void PlotHists(TFile* inFile, TFile* outFile,
+               TString treeName, TString histName, TString histTitle,
+               TString histXaxis, TString histYaxis,
+               int nBins, double xMin, double xMax,
+               TString metricToPlot, TString condition, TString dirName);
+void PlotTimeHist(TFile* inFile, TFile* outFile,
```

Comment:
Is this needed only because the current PlotHists doesn't accommodate non-uniform binning?

Reply:
That's part of it, however the real obstruction is that the data is saved as a std::map and the histogram needs to be filled differently.